packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md

```diff
@@ -17,3 +17,9 @@ When you're configuring federation between two Okta orgs, use OpenID Connect as
 1. Assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select. For instructions on how to assign the app integration to individual users and groups, see the [Assign app integrations](https://help.okta.com/okta_help.htm?id=ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
 1. Click **Save**.
 1. Copy the **Client ID** and **Client secret** from the **Client Credentials** section and paste into a text editor. You need these when you configure this Identity Provider in your other Okta org in the next section.
+
+If you plan to [enable Authentication method reference (AMR) claims](#create-an-identity-provider-in-okta) at the Okta Identity Provider (spoke org), you must enable **Use standard AMR value format** at the Okta org level (hub org) where you create the app for the spoke org. Enable this option to send AMR claim values to apps using the standard format of the supported protocol rather than the legacy Okta format.
```

Comment:
```suggestion
If you plan to [enable Authentication method reference (AMR) claims](#create-an-identity-provider-in-okta) in the Okta org (spoke org) that uses the Okta IdP, you must enable **Use standard AMR value format** in the Okta org with the Okta IdP configured (hub org). Enable this option to send AMR claim values to apps using the standard format of the supported protocol rather than the legacy Okta format.
```